KerbulatorGUI.cs

```diff
@@ -121,18 +121,10 @@ public class KerbulatorGUI {
 			this.drawMainButton = drawMainButton;
 			ChangeState(false);
 
-			functionDir = Application.persistentDataPath + "/Kerbulator";
-
-			// Sometimes, Application.persistentDataPath returns an empty string.
-			// To not completely crash, create a KerbulatorFunctions directory in the users home dir
-			if(functionDir == "/Kerbulator") {
-				string homePath =
-                    (Environment.OSVersion.Platform == PlatformID.Unix || 
-                     Environment.OSVersion.Platform == PlatformID.MacOSX)
-					 ? Environment.GetEnvironmentVariable("HOME")
-					 : Environment.ExpandEnvironmentVariables("%HOMEDRIVE%%HOMEPATH%");
-				functionDir = homePath +"/KerbulatorFunctions";
-			}
+			// Use the game base directory + PluginData as base folder for plugin data
+			functionDir = KSPUtil.ApplicationRootPath + "/PluginData" + "/Kerbulator";
+			functionDir = functionDir.Replace("\\", "/");
+			if (functionDir.EndsWith("/")) functionDir = functionDir.Substring(0, functionDir.Length - 1);
```

Comment:
How could `functionDir` ever and with `/`? Line 125 enforces that it will always end with `/Kerbulator`.